packages/cli/src/utils.ts

```diff
@@ -40,6 +40,19 @@ export const checkForSelfUpdate = async () => {
   }
 };
 
+export const checkMintedStatus = async (conn: Connection, pubAddr: string, appAddr: string, releaseAddr: string) => {
+  const results = await conn.getMultipleAccountsInfo([
+    new PublicKey(pubAddr),
+    new PublicKey(appAddr),
+    new PublicKey(releaseAddr),
+  ]);
+
+  const rentAccounts = results.filter((item) => !(item == undefined) && item?.lamports > 0);
+  if (rentAccounts?.length != 3) {
+    throw new Error("Please ensure you have minted all of your NFTs before submitting to the dApp store.");
```

Comment:
NIT: message should reference the `Solana Mobile dApp publisher portal`, to align with all the other documentation